silicone/README.md

```diff
@@ -0,0 +1,275 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+languages:
+- en
+licenses:
+- cc-by-sa-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- dialogue-modeling
+- language-modeling
+- sentiment-classification
+- topic-classification
+- semantic-similarity-scoring
+- sentiment-scoring
+---
+
+# Dataset Card for SILICONE
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** N/A
+- **Repository:** [SILICONE](https://github.com/eusip/SILICONE-benchmark)
+- **Paper:** [Hierarchical Pre-training for Sequence Labelling in Spoken Dialog](https://arxiv.org/abs/2009.11152)
+- **Leaderboard:** N/A
+- **Point of Contact:** [Ebenge Usip](ebenge.usip@telecom-paris.fr)
+
+### Dataset Summary
+
+The Sequence labellIng evaLuatIon benChmark fOr spoken laNguagE (SILICONE) benchmark is a collection of resources for training, evaluating, and analyzing natural language understanding systems specifically designed for spoken language. All datasets are in the English language and cover a variety of domains including daily life, scripted scenarios, joint task completion, phone call conversations, and televsion dialogue. Some datasets additionally include emotion and/or sentimant labels.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+English.
+
+## Dataset Structure
+
+### Data Instances
+
+#### DailyDialog Act Corpus (Dialogue Act)
+```
+{
+  'Utterance': the taxi drivers are on strike again .,
+  'Dialogue_Act': inform,
```

Comment:
```suggestion
  'Dialogue_Act': 2,  # inform
```
Here we expect to see the example as it is returned by the dataset library.
Since the `Dialogue_Act` field is a ClassLabel then it returns the label id as an integer. Here `2` corresponds to the `"inform"` label name. The correspondence is defined by the label classes list. Here it is `["commissive", "directive", "inform", "question"]`

Same for the other datasets